lib/strong.rb

```diff
@@ -0,0 +1,19 @@
+class Str
+
+  def strong_open(input)
+    input.split.each{|x| x.sub!("*", "<strong>") if x[0] == "*"}
+  end
+
+  def strong_close(input)
+    input.split.each {|x| x.sub!("*", "</strong>") if x[-2..-1].nil? != true}
+    #  && x[-2.-1].include?("*")}
+  end
+
+  def strong(input)
+    input.map!{ |x| x.include?("*") ? strong_open(x): x}
```

Comment:
Since you are doing this twice, you could make the mutating method another method and refactor this a bit 😄